docs_source_files/content/getting_started/quick.ko.md

```diff
@@ -1,23 +1,19 @@
 ---
-title: "퀵 투어"
+title: "빠르게 살펴보기"
 weight: 1
 ---
+셀레니움은 그저 하나의 도구(tool)나 API가 아니라, 여러 도구들로 구성되어 있습니다.
 
-Selenium는 하나의 툴이나 API가 아닌
-여러 툴로 구성되어있습니다.
+## 웹 드라이버(WebDriver)
 
-## WebDriver
+컴퓨터나 모바일 웹사이트 테스트를 자동화하는 것부터 시작하신다면, 아마 웹 드라이버 APIs부터 쓰게 될 겁니다.  _[웹 드라이버WebDriver]({{< ref "/webdriver/_index.md" >}})_ 는 브라우저를 조종하고 테스트를 진행할 수 있도록 제작사에서 제공하는 브라우저 자동화 API들을 사용합니다. 이는 진짜 사용자가 브라우저를 이용하는 것과 비슷합니다. 웹 드라이버 API를 애플리케이션의 코드로 컴파일할 필요가 없기 때문에, 뒤섞이지 않습니다(not intrusive). 더 나아가, 실시간으로 사용하는 것과 같은 애플리케이션을 테스트합니다.
```

Comment:
Hi @twinstae,

can you please make sure that each line width has only 70-80 charecters?

Thanks,
Harsha